strategizer/strategizer.js

```diff
@@ -220,6 +220,16 @@ export class Strategizer {
     return mergedEvaluations;
   }
 
+  static defaultOptions(ruleset) {
```

Comment:
I wanted to share these defaults, as at least an interim avoid-wildly-deviant-strategizer-behavior goal. @mmandlis had noted that eventually we'll probably want a simple way to run the strategizer on some set of recipes, and so that may further simplify what we're now doing in this PR in `outer-PEC.js`. But right now this is only the second place we've wanted to use the strategizer so I figure we should learn further before add'l factoring.

I also considered making a static `create` method instead of this, and using that in `Planner` and `OuterPEC`, but really the options were the only thing of substantial note and it doesn't look like runtime uses that static create pattern thing currently.